docs/services/matrix.md

```diff
@@ -85,5 +85,24 @@ By default, a SQL-Lite database is used, so we still need to comment it out.
 
 ```
 
-Now we are finish and we can start the container with `docker-compose up -d`
+Now you are finish and you can start the container with `docker-compose up -d`
+
+### Reset password of user 
+
+If you want to reset the password run 
+```shell
+docker-compose exec matrix -u www-data hash_password -p PASSWORD
```

Comment:
Do you really want to execute the command: `-u www-data hash_password -p PASSWORD`.
I guess you would like to execute the command: `hash_password -p PASSWORD` as `www-data`:
```shell
docker-compose exec -u www-data matrix hash_password -p PASSWORD
```